db/schema.rb

```diff
@@ -10,94 +10,96 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170309234431) do
+ActiveRecord::Schema.define(version: 2019_02_06_213404) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "categories", force: :cascade do |t|
-    t.string   "title"
-    t.text     "notes"
+    t.string "title"
+    t.text "notes"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
   end
 
   create_table "meals", force: :cascade do |t|
-    t.text     "name"
-    t.float    "calorie"
-    t.float    "protein"
-    t.float    "carb"
-    t.float    "fat"
-    t.float    "sodium"
-    t.float    "sugar"
-    t.float    "gram"
+    t.text "name"
+    t.float "calorie"
+    t.float "protein"
+    t.float "carb"
+    t.float "fat"
+    t.float "sodium"
+    t.float "sugar"
+    t.float "gram"
```

Comment:
none related schemas of this PR should not be touched. Only these two are needed for this PR https://github.com/rajgurung/keepmefiit.com/pull/22/files#diff-1acd2e7e27a227829d5d14a91c863bb6R83

Reply:
on clicking the link, i could not differentiate, which two you are talking about.